README.md

```diff
@@ -81,7 +81,7 @@ Note: The dates should be in UTC (same as GMT)
 
 # How to Use
 
-You would want to implement the delegate functions inside your view controller as they appear in the example project.
+You would want to implement the delegate functions inside your view controller as they appear in the example project.  Don't forget to assign your class as the `CalendarView` delegate.

```

Comment:
Yeah, that boggled me for a minute in the beginning.  The example project set me straight